CpManager.lua

```diff
@@ -829,13 +829,15 @@ end;
 function CpManager:minuteChanged()
 end;
 
+--FieldScan startup dialog and github disclaimer
 function CpManager:showYesNoDialogue(title, text, callbackFn)
 	-- don't show anything if the tutorial dialog is open (it takes a while until is isOpen shows true after startup, hence the clock)
 	--courseplay.debugFormat(12, "clock %d %s", courseplay.clock, tostring(g_gui.guis.YesNoDialog.target and g_gui.guis.YesNoDialog.target.isOpen))
 	if courseplay.clock < 2000 or (g_gui.guis.YesNoDialog.target and g_gui.guis.YesNoDialog.target.isOpen) then
 		return
 	end
 	--courseplay.debugFormat(12, text)
+	local text =string.format("%s\n %s",courseplay:loc('COURSEPLAY_YES_NO_FIELDSCAN'),courseplay:loc('COURSEPLAY_YES_NO_FIELDSCAN_DISCLAIMER')) 
```

Comment:
Could we rename COURSEPLAY_YES_NO_FIELDSCAN_DISCLAIMER to COURSEPLAY_SUPPORT_INFO or something, the text has nothing to do with field scanning (we just put it into the same dialog) and isn't really a disclaimer either :)